azure_pipelins/changelogs.yml

```diff
@@ -0,0 +1,84 @@
+trigger: none # Non-PR branches
```

Comment:
This pull request does not update CHANGELOG.TXT.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.TXT or resolve this comment.
<!-- changelog-comment -->